aml-ai/listLocations.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const main = async () => {
+  // [START antimoneylaunderingai_list_locations]
+  // Import google-auth-library for authentication.
+  const {GoogleAuth} = require('google-auth-library');
+
+  const listLocations = async () => {
+    const auth = new GoogleAuth({
+      scopes: 'https://www.googleapis.com/auth/cloud-platform',
+      headers: {'Content-Type': 'application/json; charset=utf-8'},
+    });
+    // TODO(developer): update the project ID as needed
+    const projectId = process.env.GOOGLE_CLOUD_PROJECT;
+    const url = `https://financialservices.googleapis.com/v1/projects/${projectId}/locations`;
+    const client = await auth.getClient();
+    let response;
+    try {
+      response = await client.request({url, method: 'GET'});
+    } catch (err) {
+      throw new Error(`API request failed: ${err}`);
+    }
+
+    console.log(JSON.stringify(response.data));
+  };
+
+  return await listLocations();
+  // [END antimoneylaunderingai_list_locations]
+};
+
+// node listLocations.js
+main().catch(err => {
+  console.error(err);
```

Comment:
Changed this so reason for failure is printed ([log()](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/actions/runs/10874211076/job/30171238388?pr=3809) vs. [error()](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/actions/runs/10874531886/job/30171912680?pr=3809)).

Reply:
cl/674939991 is merged, waiting on API enablement now.

Reply:
Hi @irataxy The API was enabled (internal: cl/676595619). Looks like a few conflicts must be resolved to get this merged, however. Do you have bandwidth to follow up, or would you like to convert this to a draft or close it for now?

Reply:
Converted to draft for now

Reply:
@irataxy The PR to enable the financialservices api was merged. I'm going to take this out of draft mode and rerun tests to confirm.

Reply:
Per @irataxy, proper client libraries for this are coming soon. Closing for now. @irataxy Feel free to reopen or create a new PR when you're ready. Thanks!